build/build.xml

```diff
@@ -397,7 +397,8 @@
 		<build-addon name="quickstart" />
 		<build-addon name="reveal" />
 		<build-addon name="saverawmessage" />
-		<build-selenium-addon />
+                <build-addon name="savexmlmessage" />
+                <build-selenium-addon />
```

Comment:
btw, this line should not be changed (same for some of the following lines).

Reply:
should I remove this change?

Reply:
Just revert the indentation change.